server/trpc/modules/index.ts

```diff
@@ -11,6 +11,7 @@ import { deadlineRouter } from './deadline/deadline.router'
 import { eventRouter } from './event/event.router'
 import { quizRouter } from './quiz/quiz.router'
 import { voucherRouter } from './voucher/voucher.router'
+import { redeemRouter } from './redemption/redemption.router'
```

Comment:
Can merge with voucher router?

Reply:
Why is there a need to merge with voucher router?

Reply:
Belongs under voucher domain.

Reply:
Redemption cannot exist without voucher